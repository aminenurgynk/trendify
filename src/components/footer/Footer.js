import React from "react";
import { Container, Typography } from "@mui/material";

const Footer = () => {
  return (
    <Container sx={{ my: 5, py: 3, borderTop: 1, borderColor: "divider" }}>
      <Typography variant="body2" color="text.secondary" align="center">
        Â© 2024 Trendify. All rights reserved.
      </Typography>
    </Container>
  );
};

export default Footer;
