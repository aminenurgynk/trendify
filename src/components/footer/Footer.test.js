// Footer.test.js

import React from 'react';
import { render, screen } from '@testing-library/react';
import Footer from './Footer';

// Unit test for the Footer component
describe('Footer component', () => {
  it('displays the correct text', () => {
    render(<Footer />);
    const footerText = screen.getByText(/Â© 2024 Trendify. All rights reserved./i);
    expect(footerText).toBeInTheDocument();
  });
});

// Snapshot test for the Footer component
it('matches the snapshot', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
