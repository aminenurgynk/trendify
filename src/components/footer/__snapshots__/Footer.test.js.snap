// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-12i9eq0-MuiContainer-root"
  >
    <p
      class="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
    >
      Â© 2024 Trendify. All rights reserved.
    </p>
  </div>
</DocumentFragment>
`;
